### PR TITLE
Update outdated files in the dev-1.19-ko.4 branch

### LINKED ISSUE #24613: Outdated files in the dev-1.19-ko.4 branch
# This is a Bug Report
## Problem
Outdated files in the dev-1.19-ko.4 branch.

### 47 files to be modified 
1. [ ] content/en/_index.html 1 file changed, 3 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/_index.md 1 file changed, 2 insertions(+)
1. [ ] content/en/docs/concepts/architecture/control-plane-node-communication.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/architecture/nodes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/configuration/configmap.md 1 file changed, 49 insertions(+), 33 deletions(-)
1. [ ] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 8 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/containers/container-lifecycle-hooks.md 1 file changed, 18 insertions(+), 7 deletions(-)
1. [ ] content/en/docs/concepts/containers/runtime-class.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md 1 file changed, 3 insertions(+), 12 deletions(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/operator.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/overview/_index.md 1 file changed, 2 insertions(+)
1. [ ] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 13 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/overview/what-is-kubernetes.md 1 file changed, 2 insertions(+)
1. [ ] content/en/docs/concepts/overview/working-with-objects/namespaces.md 1 file changed, 3 deletions(-)
1. [ ] content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/dns-pod-service.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/ingress.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/network-policies.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/service.md 1 file changed, 4 insertions(+)
1. [ ] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 39 insertions(+)
1. [ ] content/en/docs/concepts/workloads/_index.md 1 file changed, 50 insertions(+)
1. [ ] content/en/docs/concepts/workloads/controllers/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 4 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/init-containers.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 12 insertions(+), 10 deletions(-)
1. [ ] content/en/docs/contribute/_index.md 1 file changed, 7 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/contribute/review/for-approvers.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/access-authn-authz/authorization.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/glossary/kubelet.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/reference/using-api/client-libraries.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/setup/best-practices/multiple-zones.md 1 file changed, 110 insertions(+), 370 deletions(-)
1. [ ] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 62 insertions(+), 61 deletions(-)
1. [ ] content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/configure-pod-container/configure-pod-initialization.md 1 file changed, 6 insertions(+), 9 deletions(-)
1. [ ] content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md 1 file changed, 12 insertions(+), 15 deletions(-)
1. [ ] content/en/docs/tasks/extend-kubectl/kubectl-plugins.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 1 insertion(+), 5 deletions(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 6 insertions(+), 6 deletions(-)
1. [ ] content/en/docs/tasks/tools/_index.md 1 file changed, 28 insertions(+), 20 deletions(-)
1. [ ] content/en/docs/tasks/tools/install-kubectl.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/hello-minikube.md 1 file changed, 3 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/examples/application/guestbook/redis-master-service.yaml 1 file changed, 2 insertions(+), 1 deletion(-)
1. [ ] content/en/examples/pods/init-containers.yaml 1 file changed, 1 insertion(+), 1 deletion(-)

### 1 files to be renamed  
1. [ ] R100 content/en/docs/concepts/configuration/pod-overhead.md -> content/en/docs/concepts/scheduling-eviction/pod-overhead.md

### 2 files to be deleted 
1. [ ] content/en/docs/sitemap.md
1. [ ] content/en/docs/tasks/tools/install-minikube.md

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `dev-1.19-ko.4` branch.

For example:
```
# checkout `dev-1.19-ko.4`
...
# check what is updated in the upstream 
git diff dev-1.19-ko.3 dev-1.19-ko.4 -- content/en/_index.html
# apply changes to content/ko
vi content/ko/_index.html
...
# commit and push
...
# make PR to `dev-1.19-ko.4`
```

## Pages to Update

/language ko

### PR DESCRIPTION
**Pull request by pjhwa**
Update outdated files in the dev-1.19-ko.4 branch.

41 files to be modified
- [x] content/en/_index.html 1 file changed, 3 insertions(+), 1 deletion(-)
- [x] content/en/docs/_index.md 1 file changed, 2 insertions(+)
- [x] content/en/docs/concepts/configuration/configmap.md 1 file changed, 49 insertions(+), 33 deletions(-)
- [x] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 8 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/containers/container-lifecycle-hooks.md 1 file changed, 18 insertions(+), 7 deletions(-)
- [x] content/en/docs/concepts/containers/runtime-class.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md 1 file changed, 3 insertions(+), 12 deletions(-)
- [x] content/en/docs/concepts/extend-kubernetes/operator.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/overview/_index.md 1 file changed, 2 insertions(+)
- [x] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 13 insertions(+), 4 deletions(-)
- [x] content/en/docs/concepts/overview/what-is-kubernetes.md 1 file changed, 2 insertions(+)
- [x] content/en/docs/concepts/overview/working-with-objects/namespaces.md 1 file changed, 3 deletions(-)
- [x] content/en/docs/concepts/services-networking/dns-pod-service.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/services-networking/ingress.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/service.md 1 file changed, 4 insertions(+)
- [x] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 39 insertions(+)
- [x] content/en/docs/concepts/workloads/_index.md 1 file changed, 50 insertions(+)
- [x] content/en/docs/concepts/workloads/controllers/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/workloads/pods/init-containers.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 12 insertions(+), 10 deletions(-)
- [x] content/en/docs/contribute/_index.md 1 file changed, 7 insertions(+), 2 deletions(-)
- [x] content/en/docs/contribute/review/for-approvers.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/access-authn-authz/authorization.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/glossary/kubelet.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/reference/using-api/client-libraries.md 1 file changed, 1 insertion(+)
- [x] content/en/docs/setup/best-practices/multiple-zones.md 1 file changed, 110 insertions(+), 370 deletions(-)
- [x] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 62 insertions(+), 61 deletions(-)
- [x] content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/configure-pod-container/configure-pod-initialization.md 1 file changed, 6 insertions(+), 9 deletions(-)
- [x] content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md 1 file changed, 12 insertions(+), 15 deletions(-)
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 1 insertion(+), 5 deletions(-)
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 6 insertions(+), 6 deletions(-)
- [x] content/en/docs/tasks/tools/_index.md 1 file changed, 28 insertions(+), 20 deletions(-)
- [x] content/en/docs/tasks/tools/install-kubectl.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tutorials/hello-minikube.md 1 file changed, 3 insertions(+), 1 deletion(-)
- [x] content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/examples/application/guestbook/redis-master-service.yaml 1 file changed, 2 insertions(+), 1 deletion(-)
- [x] content/en/examples/pods/init-containers.yaml 1 file changed, 1 insertion(+), 1 deletion(-)

1 file is renamed
- [x] content/en/docs/concepts/configuration/pod-overhead.md -> content/en/docs/concepts/scheduling-eviction/pod-overhead.md

2 files are deleted
- [x] content/en/docs/sitemap.md
- [x] content/en/docs/tasks/tools/install-minikube.md

N files are already updated in Korean documents
- [x] content/en/docs/concepts/architecture/control-plane-node-communication.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/architecture/nodes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/network-policies.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 4 insertions(+), 4 deletions(-)
- [x] content/en/docs/tasks/extend-kubectl/kubectl-plugins.md 1 file changed, 1 insertion(+), 1 deletion(-)

Fixes #24613

### CODE REVIEW

**Comment by seokho-son** [comment]
I am working on this.

**Review by seokho-son — COMMENTED** [review]
@pjhwa Thank you for maintaining Korean l10n work branch !!
I have some minor comments on this PR !

File: content/ko/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -52,34 +52,47 @@ Angular와 같이, 컴포넌트 라이프사이클 훅을 가진 많은 프로
 
 ### 훅 핸들러 실행
 
-컨테이너 라이프사이클 관리 훅이 호출되면, 
+컨테이너 라이프사이클 관리 훅이 호출되면,
 쿠버네티스 관리 시스템은 해당 훅이 등록된 컨테이너에서 핸들러를 실행한다.
+쿠버네티스 관리 시스템은 훅 동작에 따라 핸들러를 실행하고,
+`exec` 와 `tcpSocket` 은 컨테이너에서 실행되고, `httpGet` 은 kubelet 프로세스에 의해 실행된다.
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/containers/container-lifecycle-hooks.md:58]: 저 라인을 지워야 했었는데... ^^
반영하겠습니다.

File: content/ko/docs/concepts/containers/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -52,34 +52,47 @@ Angular와 같이, 컴포넌트 라이프사이클 훅을 가진 많은 프로
...
 
-훅 핸들러 호출은 해당 컨테이너를 포함하고 있는 파드의 맥락과 동기적으로 동작한다. 
-이것은 `PostStart` 훅에 대해서, 
+훅 핸들러 호출은 해당 컨테이너를 포함하고 있는 파드의 맥락과 동기적으로 동작한다.
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/containers/container-lifecycle-hooks.md:59]: 반영하겠습니다.

File: content/ko/docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -168,6 +168,45 @@ spec:
...
 
+### 퍼시스턴트볼륨 예약
+
+컨트롤 플레인은 클러스터에서 [퍼시스턴트볼륨클레임을 일치하는 퍼시스턴트볼륨에 바인딩](#바인딩)할
+수 있다. 그러나, PVC를 특정 PV에 바인딩하려면, 미리 바인딩해야 한다.
+
+퍼시스턴트볼륨클레임에서 퍼시스턴트볼륨을 지정하여, 특정 PV와 PVC 간의 바인딩을 선언한다.
+퍼시스턴트볼륨이 존재하고 `claimRef` 필드를 통해 퍼시스턴트볼륨클레임을 예약하지 않은 경우, 퍼시스턴트볼륨 및 퍼시스턴트볼륨클레임이 바인딩된다.
+
+바인딩은 노드 선호도를 포함하여 일부 볼륨 일치 기준에 관계없이 발생한다.
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/storage/persistent-volumes.md:179]: 반영하겠습니다.

File: content/ko/docs/concepts/workloads/_index.md [hunk 4]
```diff
@@ -3,4 +3,55 @@ title: "워크로드"
 weight: 50
 description: >
   쿠버네티스에서 배포할 수 있는 가장 작은 컴퓨트 오브젝트인 파드와, 이를 실행하는 데 도움이 되는 하이-레벨(higher-level) 추상화
+no_list: true
 ---
+
+{{< glossary_definition term_id="workload" length="short" >}}
+워크로드가 단일 컴포넌트이거나 함께 작동하는 여러 컴포넌트이든 관계없이, 쿠버네티스에서는 일련의
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/workloads/_index.md:10]: 반영하겠습니다.

File: content/ko/docs/concepts/workloads/_index.md [hunk 5]
```diff
@@ -3,4 +3,55 @@ title: "워크로드"
...
+[파드](/ko/docs/concepts/workloads/pods) 집합 내에서 실행한다.
+쿠버네티스에서 파드는 클러스터에서 실행 중인 {{< glossary_tooltip text="컨테이너" term_id="container" >}}
+집합을 나타낸다.
+
+파드에는 정의된 라이프사이클이 있다. 예를 들어, 일단 파드가 클러스터에서 실행되고
+해당 파드가 실행 중인 {{< glossary_tooltip text="노드" term_id="node" >}}에서
+심각한 오류가 발생하게 되면 해당 노드의 모든 파드가 실패한다. 쿠버네티스는 이 수준의 실패를
+최종적으로 처리한다. 나중에 노드가 복구되더라도 새 파드를 만들어야 한다.
+
+그러나, 작업을 훨씬 쉽게하기 위해 각 파드를 직접 관리할 필요는 없다.
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/workloads/_index.md:20]: 반영하겠습니다.

File: content/ko/docs/concepts/workloads/_index.md [hunk 6]
```diff
@@ -3,4 +3,55 @@ title: "워크로드"
...
+* 스토리지 드라이버 또는 네트워크 플러그인과 같은 노드-로컬 기능을 제공하는
+  파드를 실행하기 위한 [데몬셋(DaemonSet)](/ko/docs/concepts/workloads/controllers/daemonset/)
+* 완료될 때까지 실행되는 작업에 대한
+  [잡(Job)](/ko/docs/concepts/workloads/controllers/job/) 및
+  [크론잡(CronJob)](/ko/docs/concepts/workloads/controllers/cronjob/)
+
+관련성을 찾을 수 있는 두 가지 지원 개념도 있다.
+* [가비지(Garbage) 수집](/ko/docs/concepts/workloads/controllers/garbage-collection/)은 _소유하는 리소스_ 가
+  제거된 후 클러스터에서 오브젝트를 정리한다.
+* [_완료된 후 TTL(time-to-live)_ 컨트롤러](/ko/docs/concepts/workloads/controllers/ttlafterfinished/)가
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/workloads/_index.md:40]: 반영하겠습니다.

File: content/ko/docs/setup/best-practices/multiple-zones.md [hunk 7]
```diff
@@ -1,398 +1,140 @@
...
+같은 워크로드 리소스의 파드 템플릿에 [노드 셀렉터 제약 조건](/ko/docs/concepts/scheduling-eviction/assign-pod-node/#노드-셀렉터-nodeselector)을
+적용할 수 있다.
+
+## 영역에 대한 스토리지 접근
+
+퍼시스턴트 볼륨이 생성되면, `PersistentVolumeLabel`
+[어드미션 컨트롤러](/docs/reference/access-authn-authz/admission-controllers/)는
+특정 영역에 연결된 모든 퍼시스턴트볼륨(PersistentVolume)에 영역 레이블을 자동으로
+추가한다. 그런 다음 {{< glossary_tooltip text="스케줄러" term_id="kube-scheduler" >}}는
+`NoVolumeZoneConflict` 술어를 통해 주어진 퍼시스턴트볼륨을 요구하는 파드가
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/setup/best-practices/multiple-zones.md:98]: 반영하겠습니다.

File: content/ko/docs/setup/best-practices/multiple-zones.md [hunk 8]
```diff
@@ -1,398 +1,140 @@
...
+
+해당 클래스의 스토리지가 사용할 수 있는 장애 도메인(영역)을 지정하는
+퍼시스턴트볼륨클레임(PersistentVolumeClaims)에 대한
+{{< glossary_tooltip text="스토리지클래스(StorageClass)" term_id="storage-class" >}}를 지정할 수 있다.
+장애 도메인 또는 영역을 인식하는 스토리지클래스 구성에 대한 자세한 내용은
+[허용된 토폴로지](/ko/docs/concepts/storage/storage-classes/#허용된-토폴로지)를 참고한다.
+
+## 네트워킹
+
+자체적으로, 쿠버네티스에는 영역 인식 네트워킹이 포함되지 않는다.
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/setup/best-practices/multiple-zones.md:109]: 반영하겠습니다.

File: content/ko/docs/setup/best-practices/multiple-zones.md [hunk 9]
```diff
@@ -1,398 +1,140 @@
...
+사용하여 클러스터 네트워킹을 구성할 수 있으며, 해당 네트워크 솔루션에는 영역별 요소가
+있을 수 있다. 예를 들어, 클라우드 제공자가
+`type=LoadBalancer` 를 사용하여 서비스를 지원하는 경우, 로드 밸런서는 지정된 연결을 처리하는
+로드 밸런서 요소와 동일한 영역에서 실행 중인 파드로만 트래픽을 보낼 수 있다.
+자세한 내용은 클라우드 제공자의 문서를 확인한다.
+
+사용자 정의 또는 온-프레미스 배포의 경우, 비슷한 고려 사항이 적용된다.
+다른 장애 영역 처리를 포함한 {{< glossary_tooltip text="서비스" term_id="service" >}}와
+{{< glossary_tooltip text="인그레스(Ingress)" term_id="ingress" >}} 동작은
+클러스터가 정확히 설정된 방식에 따라 달라진다.
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/setup/best-practices/multiple-zones.md:120]: 반영하겠습니다.

**Comment by pjhwa** [comment]
@seokho-son 빠른 리뷰 감사합니다.
문장을 자연스럽게 다듬어주셔서 감사합니다. 👍 

의견주신 내용은 모두 반영하여 업데이트하였습니다.

**Comment by seokho-son** [comment]
감사합니다!
/lgtm

**Pull request merged**